addons/project/controllers/portal.py

```diff
@@ -23,7 +23,7 @@ def _prepare_home_portal_values(self, counters):
             values['project_count'] = request.env['project.project'].search_count([]) \
                 if request.env['project.project'].check_access_rights('read', raise_exception=False) else 0
         if 'task_count' in counters:
-            values['task_count'] = request.env['project.task'].sudo().search_count([('project_id', '!=', False), ('message_partner_ids', 'in', request.env.user.partner_id.ids), ('project_privacy_visibility', '=', 'portal')]) \
+            values['task_count'] = request.env['project.task'].search_count([])\
```

Comment:
```suggestion
            values['task_count'] = request.env['project.task'].search_count([('project_id', '!=', False)])\
```
we could let `[('project_id', '!=', False)]` to be sure we will not take into account the private tasks (in the case an internal user goes to `/my` route).

Reply:
sure !